README.md

```diff
@@ -0,0 +1,41 @@
+[![GoDoc](https://godoc.org/github.com/ooni/probe-engine?status.svg)](https://godoc.org/github.com/ooni/probe-engine) [![Build Status](https://travis-ci.org/ooni/probe-engine.svg?branch=master)](https://travis-ci.org/ooni/probe-engine) [![Coverage Status](https://coveralls.io/repos/github/ooni/probe-engine/badge.svg?branch=master)](https://coveralls.io/github/ooni/probe-engine?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/ooni/probe-engine)](https://goreportcard.com/report/github.com/ooni/probe-engine)
+
+# OONI probe measurement engine
+
+This repository contains OONI probe's [measurement engine](
+https://github.com/ooni/spec/tree/master/probe#engine). That is, the
+piece of software that implements OONI nettests.
+
+## API
+
+You can [browse ooni/probe-engine's API](
+https://godoc.org/github.com/ooni/probe-engine?status.svg)
+at [godoc.org](https://godoc.org/). We don't provide any API
+stability guarantees and, as such, we will never release v1.0.0
+of this software.
```

Comment:
This sounds like a pretty strong statement, perhaps we can just say that currently we don't provide any sort of API stability.

We may still want at some point to cut a 1.0.0 even if it's just to communicate some level of API compatibility for our own purpose.

Reply:
> perhaps we can just say that currently we don't provide any sort of API stability.

Sounds less strong, okay